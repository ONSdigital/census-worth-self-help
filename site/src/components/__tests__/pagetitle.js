import React from "react"
import renderer from "react-test-renderer"

import PageTitle from "../pagetitle"

describe("PageTitle", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<PageTitle>This is a title</PageTitle>)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})