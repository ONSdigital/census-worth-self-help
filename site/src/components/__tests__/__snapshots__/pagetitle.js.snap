// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTitle renders correctly 1`] = `
<div
  css={
    Object {
      "map": undefined,
      "name": "12ew49b",
      "next": undefined,
      "styles": "
        
    padding: 5px 15px;
  
      ",
    }
  }
>
  <h1
    className="Article-Title-Style"
    css={
      Object {
        "map": undefined,
        "name": "1xcgh0k",
        "next": undefined,
        "styles": "
          margin-bottom: 0px;
          display: flex;
        ",
      }
    }
    data-testid="search-result-title"
  >
    This is a title
  </h1>
</div>
`;
